dist/rules/exports-require-release-tags.js

```diff
@@ -0,0 +1,247 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+// Writing all of these eslint rules in javascript so we can run them before the build step
+
+"use strict";
+
+const { getParserServices } = require("./utils/parser");
+const ts = require("typescript");
+const fs = require("fs");
+
+/** converts the numeric typescript enum value for ts.SyntaxKind to a string. Defaults to "real". */
+const getSyntaxKindFriendlyName = (syntaxKind) => {
+  const syntaxKindFriendlyNames = {
+    [ts.SyntaxKind.ClassDeclaration]: "real",
```

Comment:
why is ClassDeclaration friendly name `real`? and why is that the default. I'm not sure I understand

Reply:
I guess on second thought, you didn't originally write this, but do you happen to know? genuinely curious

Reply:
I was also mystified by this. My best guess is `class` caused an issue somewhere as a reserved word and they went with... real? Commit history obscured as repo was moved from core. Looking into it as I am very curious.

Reply:
No comment on the PR... @aruniverse remember anything about this?

Reply:
ya, weird. you can resolve this thread at your discretion